### ISSUE TITLE
Package version issues

### ISSUE THREAD

**Issue by Faran52** [linuxdeepin/developer-center#608]
The version of some packages keeps downgrading on apt-get full-upgrade

such as Skype 8.30 -> 8.11
elasticsearch 6.x -> 1.x
chrome 69.x -> 68.x
slack 3.3.1 -> 2.x 
and more...

can you please update the packages or is there a way I can choose to ignore the official repo packages...

**Comment by BLumia** [comment]
Related preference file: `/etc/apt/preferences.d/deepin`

You can also use `apt-mark hold <package_name>` to mark the package you would like to keep.

**Comment by Faran52** [comment]
yes that is an option I'm using for npw, but can the repo please be updated or I can force packages from 3rd party repos like chrome putting packages on hold would mean I cannot update them from their official repos like skype, chrome, slack etc

**Comment by aiamuzz** [comment]
Yes ... even i have been facing issues with old versions on Deepin repo's ... for instance i need the latest OStree package to run the latest Flatpak directly from source ... but because deepin repo has an old version ... i couldn't upgrade the same.

trying to upgrade a package from a third party place throws up a lot of dependency issues ... attempting to upgrade those dependencies is increasing the problem exponentially ... like a chain dependency problem.

I wish there was an easier way to upgrade our personal favorites through a third party sources with no dependency issues ... don't all packages are designed to have backward compatibility ? ... i have come across more than a few instances where the dependency forces us to downgrade certain packages ...

**Comment by BLumia** [comment]
> can you plz explain a bit about the preference file

Read [AptPreferences](https://wiki.debian.org/AptPreferences), then you'll know what to do. You can also simply remove `/etc/apt/preferences.d/deepin` if you don't want we force to use the package provided by the offical repo.

The reason we do this is, there are a lot of bug report caused by user who added 3rd party repo and install package from there which may break the package dependencie in some case. So if you are planing to remove that preference file, **ensure you know what actually you are doing**, if not, read the related manual first.
